wetterdienst/dwd/radar/store.py

```diff
@@ -5,56 +5,64 @@
 
 from wetterdienst import TimeResolution
 from wetterdienst.dwd.metadata.datetime import DatetimeFormat
+from wetterdienst.dwd.radar.metadata import RadarParameter
 
 
-def store_radolan_data(
+def store_radar_data(
+    parameter: RadarParameter,
     date_time_and_file: Tuple[datetime, BytesIO],
     time_resolution: TimeResolution,
     folder: Union[str, Path],
-) -> None:
+) -> None:  # pragma: no cover
+    """
+    Stores a binary file of radar data locally.
+    """
 
     date_time, file = date_time_and_file
 
-    filepath = build_local_filepath_for_radolan(date_time, folder, time_resolution)
+    filepath = _build_local_filepath(parameter, date_time, folder, time_resolution)
 
     filepath.parent.mkdir(parents=True, exist_ok=True)
```

Comment:
This will not work in case that the output of _build_local_filepath is a Union[str, Path].

Reply:
Thanks. `radar/store.py` is currently completely defunct.